src/index.d.ts

```diff
@@ -19,3 +19,83 @@ declare namespace dynamodb {
   ): void;
   export function dynamoDBTestHooks(useUniqueTables?: boolean): Hooks;
 }
+
+declare namespace lambda {
+  interface Environment {
+    [key:string]: string | null;
```

Comment:
I think `Record<string, string>` would be best here.

Reply:
Wouldn't that make it unable to be `null`? This can be null if we want to delete attributes from the environment

Reply:
`delete` will remove the attribute binding though. Environment variables can only be represented as strings, a key miss would result in undefined.

Reply:
What are you referring to when you say `delete`? I know they can only be represented as strings, but when we pass in environments from functions it uses the `createEnvironmentVariables` function which expects keys that need to be deleted to be provided with `null` values.

Reply:
Oh, I misunderstood the use case